molecule/default/molecule.yml

```diff
@@ -67,6 +74,35 @@ provisioner:
       # space and the slash.  This option converts those characters to
       # underscores when creating the group name.
       force_valid_group_names: always
+  inventory:
+    host_vars:
+      debian9:
```

Comment:
It's not being tested, but to fit with needing systemd this should probably be `debian9_systemd`.

Reply:
See commit fc9794f.

Reply:
Guess I should have said something similar about this in `ansible-role-kali` as well x)

Reply:
No worries.  I'm going to make sure all the Ansible roles are correct.

Thanks for catching this.  It could have led to some build failures that would be difficult to debug.